doc/alpha_shape/pgr_alphaShape.rst

```diff
@@ -26,16 +26,16 @@ pgr_alphaShape
 * Renamed in v2.0.0
 
 **Supported versions:**
-current(`3.0 <http://docs.pgrouting.org/3.0/en/pgr_alphaShape.html>`__)
+current(`3.0 <http://docs.pgrouting.org/dev/en/pgr_alphaShape.html>`__)
 `2.6 <http://docs.pgrouting.org/2.6/en/pgr_alphaShape.html>`__
 `2.5 <http://docs.pgrouting.org/2.5/en/pgr_alphaShape.html>`__
-
-**Unsupported versions:**
 `2.4 <http://docs.pgrouting.org/2.4/en/pgr_alphaShape.html>`__
-`2.3 <http://docs.pgrouting.org/2.3/en/src/allpairs/doc/pgr_alphaShape.html>`__
-`2.2 <http://docs.pgrouting.org/2.2/en/src/allpairs/doc/pgr_alphaShape.html>`__
-`2.1 <http://docs.pgrouting.org/2.2/en/src/allpairs/doc/pgr_alphaShape.html>`__
 
+**Unsupported versions:**
+`2.3 <http://docs.pgrouting.org/2.3/en/src/alpha_shape/doc/pgr_alphaShape.html#pgr-alphaShape>`__
```

Comment:
Can you remove the "#pgr-alphaShape" from the links, otherwise you are not sending to the page, but to a title inside the page

should be:  up to where its the html
`2.3 <http://docs.pgrouting.org/2.3/en/src/alpha_shape/doc/pgr_alphaShape.html>`__

The same with all the links please.

Reply:
ok